packages/experience-builder-sdk/src/utils/withComponentWrapper.tsx

```diff
@@ -40,8 +38,11 @@ export function withComponentWrapper<T>(
       ToolTipAndPlaceholder,
       ...restOfDragProps
     } = dragProps;
+
+    const Tag = (options.wrapContainer || options.wrapContainerTag || 'div') as string;
```

Comment:
Maybe we could just cleanup the `wrapContainerTag` instead of supporting it? As far as I can see it's marked as deprecated and wasn't working until now, so no one is relying on it.